katacomb/katacomb/conf/parameters/MKAT_wide_L.yaml

```diff
@@ -8,8 +8,8 @@
 # Further details of the parameter selection can be found in JIRA ticket
 # SPR1-879: https://skaafrica.atlassian.net/browse/SPR1-879
```

Comment:
This ticket does seem to say anything about the L-band, please verify this.

Reply:
It does - in that it compares UHF and S band parameters to L band parameters. There is no ticket describing the L-band parameters alone.